app/router.cjsx

```diff
@@ -187,15 +189,17 @@ module.exports =
       </Route>
       <Route path="collaborators" component={require './pages/lab/collaborators'} />
       <Route path="media" component={EditMediaPage} />
-      <Route path="workflow/:workflowID" component={require './pages/lab/workflow'} />
+      <Route path="workflows/:workflowID" component={require './pages/lab/workflow'} />
```

Comment:
We need a redirect in place if and when routes are changed in case someone has a bookmark to their lab pages.